src/test/app/Discrepancy_test.cpp

```diff
@@ -24,122 +24,124 @@
 #include <ripple/beast/core/LexicalCast.h>
 #include <ripple/protocol/JsonFields.h>
 #include <ripple/protocol/SField.h>
+#include <ripple/ledger/tests/PathSet.h>
```

Comment:
It looks like you can remove a number of these #includes.  Using clang I was able to remove:

- ripple/test/jtx/Env.h
- ripple/beast/unit_test.h
- ripple/protocol/SField.h
- boost/filesystem.hpp
- boost/scope_exit.hpp
-  ripple/basics/StringUtilities.h
- fstream

Not mandatory, just for your consideration.